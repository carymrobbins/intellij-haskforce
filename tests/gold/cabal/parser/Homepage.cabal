homepage:       http://dingske.github.com

bug-reports:    https://github.com/dingske/dongske/issues

maintainer: koekoek.merel@nest.com
            hier nog ewa meer