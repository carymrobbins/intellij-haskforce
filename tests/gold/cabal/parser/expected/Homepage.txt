Cabal File
  KEY_OR_CONFIG
    KEY
      HOMEPAGE
        PsiElement(CabalTokenType.homepage)('homepage')
        PsiElement(CabalTokenType.:)(':')
        URL
          PsiElement(CabalTokenType.freeformRegexp)('http://dingske.github.com')
  KEY_OR_CONFIG
    KEY
      BUG_REPORTS
        PsiElement(CabalTokenType.bug-reports)('bug-reports')
        PsiElement(CabalTokenType.:)(':')
        URL
          PsiElement(CabalTokenType.freeformRegexp)('https://github.com/dingske/dongske/issues')
  KEY_OR_CONFIG
    KEY
      MAINTAINER
        PsiElement(CabalTokenType.maintainer)('maintainer')
        PsiElement(CabalTokenType.:)(':')
        ADDRESS
          PsiElement(CabalTokenType.freeformRegexp)('koekoek.merel@nest.com')
          PsiElement(CabalTokenType.freeformRegexp)('hier nog ewa meer')